spyder/plugins/editor/widgets/codeeditor/lsp_mixin.py

```diff
@@ -255,7 +257,8 @@ def start_completion_services(self):
             additional_msg = "cloned editor"
         else:
             additional_msg = ""
-            self.document_did_open()
+
+        self.document_did_open()
```

Comment:
```suggestion
            self.document_did_open()
```

This needs to remain as it was before.